src/components/AnimatePresence/PresenceChild.tsx

```diff
@@ -39,11 +39,8 @@ export const PresenceChild = ({
             custom,
             onExitComplete: (childId: number) => {
                 presenceChildren.set(childId, true)
-                let allComplete = true
-                presenceChildren.forEach((isComplete) => {
-                    if (!isComplete) allComplete = false
-                })
-
+                const completed = [...presenceChildren.values()]
```

Comment:
Doesn't values() already return a new array?

Reply:
No, it returns a `MapIterator`, which behaves like an array in some cases but doesn't have `forEach`, `map`, `filter`, `some`, `every`, etc.